patterns/header-centered-search-pattern.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Title: WooCommerce Header Centered Search
+ * Slug: woocommerce-blocks/header-centered-search
+ * Categories: WooCommerce
+ * Block Types: core/template-part/header
+ */
+?>
+
+<!-- wp:group {"align":"full","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull">
+	<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"right":"2%","bottom":"16px","left":"2%","top":"16px"},"margin":{"top":"0px","bottom":"0px"}}},"className":"sticky-header","layout":{"type":"flex","flexWrap":"wrap","justifyContent":"space-between"}} -->
+	<div class="wp-block-group alignfull sticky-header" style="margin-top:0px;margin-bottom:0px;padding-top:16px;padding-right:2%;padding-bottom:16px;padding-left:2%">
+		<!-- wp:group {"style":{"spacing":{"blockGap":"40px"}},"layout":{"type":"flex","flexWrap":"wrap"}} -->
+		<div class="wp-block-group">
+			<!-- wp:site-logo {"shouldSyncIcon":false} /-->
+			<!-- wp:site-title /-->
+		</div>
+		<!-- /wp:group -->
+
+		<!-- wp:group {"style":{"spacing":{"blockGap":"8px"},"typography":{"fontSize":"13px"}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"right"}} -->
+		<div class="wp-block-group" style="font-size:13px">
+			<!-- wp:group {"style":{"spacing":{"blockGap":"16px"}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+			<div class="wp-block-group">
+				<!-- wp:separator {"className":"is-style-wide"} -->
+				<hr class="wp-block-separator has-alpha-channel-opacity is-style-wide" />
+				<!-- /wp:separator -->
+			</div>
+			<!-- /wp:group -->
```

Comment:
The only thing I found, was that this Row with Separator inside doesn't influence the look of the template. Could they be removed or there's some other reason that I missed?

| With | <img width="1452" alt="Screen Shot 2023-04-19 at 08 58 16 AM" src="https://user-images.githubusercontent.com/20098064/232992119-1dd649bb-afb7-439f-ae6b-27aa939149ef.png"> |
| --- | --- |
| Without | <img width="1453" alt="image" src="https://user-images.githubusercontent.com/20098064/232991942-bb4912d0-5e5d-4405-8a10-3d427f95a33d.png"> |

Reply:
Thanks, removed!